mautic/mautic
# [UX] Specify redirection labels present when creating a landing page

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
 This update enriches the redirection options during landing page creation with descriptive labels. It aims to guide users in selecting the appropriate redirect type by explaining the difference between temporary and permanent redirects, and when each should be used. This clarification assists users in making informed decisions, ensuring their landing pages function as intended and enhancing the overall creation process.

Now people can just select the less complicated option to fit 99,99% of their needs.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Landing Pages
3. Create a new
4. Click to see the options for Redirection

![image](https://github.com/mautic/mautic/assets/116097999/1710dd0b-f5a9-4dfd-b0e6-0d4cc106430f)
